mypy.ini

```diff
@@ -0,0 +1,215 @@
+# Global options:
+
+[mypy]
+ignore_missing_imports = True
+python_version = 3.10
```

Comment:
No, it needs to be Python 3.10 because if it's 3.7 and we test on 3.10, type checking fails somewhere in numpy probably due to some logic they have implemented based on the Python version.